src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -75,6 +75,10 @@ object ScalafixPlugin extends AutoPlugin {
         "Optional location to .scalafix.conf file to specify which scalafix rules should run. " +
           "Defaults to the build base directory if a .scalafix.conf file exists."
       )
+
+    val scalafixCallback: SettingKey[ScalafixMainCallback] =
+      settingKey[ScalafixMainCallback]("")
```

Comment:
```suggestion
      settingKey[ScalafixMainCallback](
        "The handler for the diagnostics emitted during scalafix execution. " +
          "Defaults to a wrapper around `sbt.Logger`."
      )
```

Reply:
For the record, your PR made me realize docs were not up to date: https://github.com/scalacenter/sbt-scalafix/pull/385.